Test/commandline/silent.bpl

```diff
@@ -0,0 +1,7 @@
+// RUN: %parallel-boogie -silent "%s" > "%t"
```

Comment:
I thought it did, but apparently not! So `-silent` is not super useful for Boogie alone. The internal option it controls is useful for clients of Boogie as a library, like Dafny, however.

Reply:
How does dafny know whether boogie is successful or not? I presume it makes some API call rather than spawning a command-line invocation.

I suggest you add an issue for boogie to return a proper exit code (though it is not an issue for me) like all good citizens should

Reply:
Is the exit code typically spit out on the console along with other output?

It would be great if we could add the standard story for exit code to Boogie also.

Reply:
In dafny testing, we write things like %exits-with 1 %parallel-boogie ...
to check the output exit code